.github/workflows/integration-tests.yml

```
@@ -0,0 +1,331 @@
+name: integration-tests
+on:
+  push:
+    branches: [ "518-implement-semi-automatic-integration-tests", "642-integration-tests-add-tests-for-aliens", "648-integrationtests-add-consistency-test-for-random-putgetdelete-requests-with-bobt-tool" ] #REMOVE WHEN ON MASTER
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+(.alpha.[0-9]+)*"
+  pull_request_review:
+    types: [submitted]
+  workflow_dispatch:
+    inputs:
+      bob_nodes_amount:
+        description: 'Amount of nodes in bob cluster for tests'
+        required: true
+        default: '4'
+      bob_vdisk_count:
+        description: 'Vdisks count'
+        required: true
+        default: '4'
+      bob_replicas_amount:
+        description: 'Amount of vdisks replicas'
+        required: true
+        default: '2'
+      bob_payload_size:
+        description: 'Payload size'
+        required: true
+        default: '10240'
+      bob_record_count:
+        description: 'Amount of records to process during test'
+        required: true
+        default: '10000'
+      bob_first_index:
+        description: 'Index of the first record to proceed during tests'
+        required: true
+        default: '0'
+      bob_bobt_count:
+        description: 'Amount of iterations for chain operation test'
+        required: true
+        default: '10000'
+      bob_bobt_first_index:
+        description: 'Bottom boundry for chain test'
+        required: true
+        default: '0'
+      bob_bobt_last_index:
+        description: 'Upper boundry for chain test'
+        required: true
+        default: '100000'
+jobs:
+  build-binaries:
+    if: ${{ github.event_name != 'pull_request_review' || github.event.review.state == 'approved' }}
+    strategy:
+      matrix:
+        key: [8, 16]
+    env:
+      TARGET: "x86_64-unknown-linux-musl"
+      BUILD_MODE: "test"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checks-out repository 
+        uses: actions/checkout@v3
+      - name: Set env variables and install packages
+        run: |
+          echo "BOB_COMMIT_HASH=$(echo ${GITHUB_SHA} | cut -c1-8)" >> $GITHUB_ENV
+          echo "BOB_KEY_SIZE=${{ matrix.key }}" >> $GITHUB_ENV
+          sudo apt-get update && sudo apt-get install -y --no-install-recommends musl-tools
+      - name: Install latest stable
+        uses: actions-rs/toolchain@v1.0.6
+        with:
+          toolchain: stable
+          target: ${{ env.TARGET }}
+          default: true
+      - name: Build binaries
+        uses: actions-rs/cargo@v1.0.1
+        with:
+          command: build
+          args: --target=${{ env.TARGET }} --profile=${{ (env.BUILD_MODE == 'test' && 'dev') || env.BUILD_MODE }}
```

Why did you change comparison condition?
```suggestion
          args: --target=${{ env.TARGET }} --profile=${{ (env.BUILD_MODE == 'debug' && 'dev') || env.BUILD_MODE }}
```